docs/guides/how-to/general.md

```diff
@@ -0,0 +1,48 @@
+Here are several pointers to other places in the docs, for general or frequent questions.
+
+
+### Set a Service Name
+To see your service names in the logfire UI, you just need to set the `service_name` parameter in
+[`logfire.configure()`][logfire.configure(service_name)].
+
+### Understanding the SQL Schema
+With Logfire, you can use the UI to run arbitrary SQL queries against your trace and metric data to
+analyze and investigate your system. For details of the data model, see
+our [schema docs](../web-ui/explore.md#records-schema)
+
+### Disable Logfire During Tests
+Logfire makes it very easy to test the emitted logs and spans. See the [testing with logfire](../advanced/testing.md) docs.
+If you encounter difficulties disabling logfire during testing (and are using pytest),
+ensure that in your conftest.py you set:
+
+```py title="conftest.py"
+import logfire
+
+logfire.configure(send_to_logfire=False)
+```
+
+### Setting an Environment (e.g. dev, prod)
+With Logfire you can distinguish which environment you are sending data to. See the [environments](../advanced/environments.md) page.
+
+### Querying Logs via API
+Logfire provides a web API for programmatically running arbitrary SQL queries against the data in your Logfire projects.
+See the [Query API docs](../advanced/query-api.md)
+
+
+### Use logfire Without Sending Data to a Server
+To disable sending data to a server and just emit traces to stdout:
+```py title="main.py"
+import logfire
+
+logfire.configure(send_to_logfire=False)
```

Comment:
Similar comments to what I said about `service_name`. This feels redundant.

I'm guessing this is inspired by https://pydanticlogfire.slack.com/archives/C06EDRBSAH3/p1722341938689489

The user was asking about hosting their own backend, the end result was https://logfire.pydantic.dev/docs/guides/advanced/alternative-backends/ (both the docs and a change in behaviour) and those docs mention `send_to_logfire=False`.